### PR TITLE
[onert-micro] Add kernels for GRU models

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This pr adds kernels for GRU models.

for issue https://github.com/Samsung/ONE/issues/10465

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

File: onert-micro/CMakeLists.txt [hunk 1]
```diff
@@ -105,6 +105,11 @@ if (DIS_FLOAT)
     list(APPEND CMAKE_ARM_OPTIONS "-DDIS_FLOAT=ON")
 endif()
 
+if (DIS_DYN_SHAPES)
+    message(STATUS "ONERT-MICRO will use dynamic shapes")
```
Comment (chunseoklee) [inline onert-micro/CMakeLists.txt:109]:
```suggestion
    message(STATUS "ONERT-MICRO will not use dynamic shapes")
```

**Review by chunseoklee — CHANGES_REQUESTED** [review]

File: onert-micro/luci-interpreter/pal/mcu/pal.cmake
Comment (chunseoklee) [inline onert-micro/luci-interpreter/pal/mcu/pal.cmake:1]: Any specific reason back to 2.6.0 ?

**Review by SlavikMIPT — APPROVED** [review]
LGTM

File: onert-micro/CMakeLists.txt (hunk 1 above)
Comment (BalyshevArtem) [reply onert-micro/CMakeLists.txt]: Done

File: onert-micro/luci-interpreter/pal/mcu/pal.cmake
Comment (BalyshevArtem) [reply onert-micro/luci-interpreter/pal/mcu/pal.cmake:1]: For supporting TizenRT os purpose

Comment (chunseoklee) [reply onert-micro/luci-interpreter/pal/mcu/pal.cmake:1]: @BalyshevArtem said that kernel of 2.8.0 has issue on std lib in tizenrt

**Review by chunseoklee — APPROVED** [review]

**Review by SlavikMIPT — APPROVED** [review]

**Pull request merged**